recipes/apscheduler/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "3.0.5" %}
+
+package:
+  name: apscheduler
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/agronholm/apscheduler.git
+  git_rev: v{{ version }}
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
Sorry, should have spotted it sooner, but whenever using setuptools and conda, please ``python setup.py install --single-version-externally-managed --record=record.txt``.